_posts/en/newsletters/2020-06-17-newsletter.md

```diff
@@ -0,0 +1,193 @@
+        limits the ability to use Wasabi with user-specified amounts or
+        for making payments with arbitrary amounts.
+
+    This week, several contributors to Wasabi [posted][wabisabi post]
+    to the Bitcoin-Dev mailing list about a new
+    protocol they call WabiSabi that conceptually extends their existing protocol
+    with a technique adapted from [confidential transactions][].  This allows a
+    client to create a commitment to arbitrary output amounts and---without
+    revealing the amounts---prove that each amount is individually
+    within a specified range (e.g. 0.0001 BTC to 21 million BTC) and
+    that they collectively sum to a specified value.  The coordinator
+    uses this specified value to verify that the sum of the outputs the
+    client wants to create is equal to the sum of the inputs provided by
+    the client (minus fees).  The coordinator can then provide an
+    anonymous credential for each output commitment that allows the
+    client to later anonymously submit the revealed output to the
+    coordinator for inclusion in the transaction template.
+
+    Software that implements the protocol will be able to create
+    coordinated coinjoins---with privacy from the coordinator---that
+    allow the clients to select their output amounts, facilitating
+    experiments with non-equal value coinjoins (see [Newsletter
+    #79][news79 unequal coinjoin]) and payments made within a coinjoin
+    (either to third-parties not participating in the coinjoin or to
+    parties within a coinjoin).
+
+    The proposed protocol contains a significant number of differences
+    from Wasabi's current protocol (such as replacing blind signatures
+    with keyed-verification anonymous credentials), so its authors are
+    seeking review, criticism, and suggestions about how the protocol
+    can be used most effectively.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Caravan adds BIP32, coin selection, hardware wallet test suite:**
+  In addition to single address multisig coordination, Caravan now supports HD
+  wallet multisig coordination and coin selection features. [Unchained Capital
+  also announced][unchained caravan blog] a test suite for testing hardware
+  wallets interacting within a web browser.
+
+- **Bitpay's Copay and Bitcore projects support native segwit:**
+  Bitpay's [Copay wallet][copay segwit] and backend [Bitcore service][bitcore
+  segwit] both now support the creation of and spending from native segwit outputs.
+
+- **Desktop version of Blockstream Green wallet:**
+  Blockstream has released their [Green wallet for desktop][blockstream green desktop blog]
+  on macOS, Windows, and Linux. The desktop version supports 2-of-3 and 2-of-2
+  multisig wallets as well as Tor and testnet support.
+
+- **React native library photon-lib announced:** [Tankred Hase
```

Comment:
Perhaps set off "photon-lib" with commas. Here and line 161, I believe the mobile app JS framework is normally written as "React Native".